frontend/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Are these defaults derived from somewhere? Or explicit choices?

Reply:
This came from the turborepo starter. Though I think we should do a quick sweep to ensure the options are what we want. I have a [task](https://codedotorg.atlassian.net/browse/ACQ-2793) for creating a common tsconfig on Jira to do that.